css/footer.css

```diff
@@ -40,3 +40,25 @@
 .microsite-footer__housekeeping ul {
   list-style-type: none;
 }
+
+.region-microsites-footer h2,
+.region-microsites-footer h3,
+.region-microsites-footer h4,
+.region-microsites-footer h5,
+.region-microsites-footer h6 {
+  color: var(--footer-text-color)
+}
+
+footer .lgd-row__one-quarter,
```

Comment:
Can we use a class here, instead of an element - we might have `footer` elements later for node metadata or something like that.